soc/nxp/mcx/soc.yml

```diff
@@ -7,6 +7,9 @@ family:
       cpuclusters:
       - name: cpu0
       - name: cpu1
+  - name: mcxn23x
```

Comment:
why are we making a new series for every SOC? can we please change this so that the series is just "mcxn" and put both SOC under that series, instead of copy pasting code all over the place that will require duplicated maintenance effort